Core/EventHandlers.lua

```diff
@@ -582,6 +582,22 @@ function R:OnIslandCompleted(event, mapID, winner)
 	end
 end
 
+local Scalebane_Keep_QuestID = 70867
```

Comment:
Since this is a constant, we should probably capitalize it (for consistency's sake)?

``local SCALEBANE_KEEP_SCENARIO_COMPLETION = 70867``